src/parser/line.rs

```diff
@@ -118,15 +119,16 @@ impl<'l> Line<'l> {
 
 impl<'l> From<&'l str> for Line<'l> {
     fn from(line: &'l str) -> Self {
-        if line.is_empty() || line.starts_with(HASH) {
+        if line.is_empty() || line.starts_with(HASH) || line.eq(EXPORT) {
             return Self::Empty;
         };
 
         let mut parts = line.splitn(2, '=');
 
         match (parts.next(), parts.next()) {
             (Some(k), Some(v)) => {
-                let key = k.trim();
+                let wo_export = k.strip_prefix(EXPORT).unwrap_or(k);
+                let key = wo_export.trim();
```

Comment:
```suggestion
                let key = k.strip_prefix(EXPORT).unwrap_or(k).trim();
```

Reply:
I think we could just inline the expression as we are not using `wo_export` anywhere else :)

Reply:
Thanks for the feedback! 

I've inlined as per the suggestion in an amend commit (--force-push ed)